src/interfaces/github.ts

```diff
@@ -56,9 +56,11 @@ interface GitHubWorkflowRunHeadCommit {
 	committer: GitHubWorkflowRunAuthorAndCommitter;
 }
 
+//refer from https://docs.github.com/en/rest/repos/repos#get-a-repository
 interface GitHubWorkflowRunRepositoryOwner {
 	login: string;
 	id: number;
+	name: string | null;
```

Comment:
It has the schema as bellow from https://docs.github.com/en/rest/repos/repos#get-a-repository
```json
"owner": {
  "title": "Simple User",
  "description": "Simple User",
  "type": "object",
  "properties": {
    "name": {
      "type": [
        "string",
        "null"
      ]
    }
  }
}
```
So type it as `string | null`

Reply:
We don't use `null` in js/ts if we can help it.  this should be `name?:string`

Reply:
oh wait, is this a github rest API type?

Reply:
Yup.